### PR TITLE
feat: devices: Add EXT_SITMPUSW resolved issues to release notes

### PR DESCRIPTION
**Pull request by jsuhaas22**
Add resolved issues to released notes of AM62PX, AM62X and AM64X.

### CODE REVIEW

File: source/devices/AM62X/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -254,6 +254,8 @@ Issues Resolved
    "SYSFW-7617","LPM mode selection logic does not check for constraints on all the hosts/devices"
    "SYSFW-7739","LPM constraint messages received by TIFS not forwarded to DM"
    "SYSFW-7740","Only the last latency constraint value passed from a host considered in LPM selection"
+   "EXT_SITMPUSW-32", "SDK Makefile: Fix logic to pick key directory for mkimage"
+   "EXT_SITMPUSW-31", "IPC Firmwares missing from base images."
```
Comment (cshilwant) [inline source/devices/AM62X/linux/Release_Specific_Release_Notes.rst:258]: Drop `EXT_SITMPUSW-31`.... I see it's already covered above

File: source/devices/AM64X/linux/Release_Specific_Release_Notes.rst [hunk 2]
```diff
@@ -218,6 +218,8 @@ Issues Resolved
    "EXT_EP-12104","AM64x NAND dtso file missing in kernel 6.6"
    "SYSFW-6426","Ownership of a firewall region can be transferred to an invalid host"
    "SYSFW-6432","Set device API doesn't return Error when PD is in transition state"
+   "EXT_SITMPUSW-32", "SDK Makefile: Fix logic to pick key directory for mkimage"
+   "EXT_SITMPUSW-31", "IPC Firmwares missing from base images."
```
Comment (cshilwant) [inline source/devices/AM64X/linux/Release_Specific_Release_Notes.rst:222]: Drop EXT_SITMPUSW-31.... I see it's already covered above

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst [hunk 3]
```diff
@@ -238,6 +238,10 @@ Issues Resolved
    "SYSFW-7739","LPM constraint messages received by TIFS not forwarded to DM"
    "SYSFW-7740","Only the last latency constraint value passed from a host considered in LPM selection"
    "SYSFW-7753","A53 privid is set to 4 and does not match other SOCs value of 1"
+   "EXT_SITMPUSW-33", "Resolve GPU crash caused by Display cluster demo"
+   "EXT_SITMPUSW-32", "SDK Makefile: Fix logic to pick key directory for mkimage"
+   "EXT_SITMPUSW-31", "IPC Firmwares missing from base images."
+   "EXT_SITMPUSW-30", "Debian: Deepsleep doesn't work on AM62P"
```
Comment (cshilwant) [inline source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst:245]: EXT_SITMPUSW-29 looks missing from AM62P

**Review by cshilwant — CHANGES_REQUESTED** [review]

**Review by cshilwant — DISMISSED** [review]

File: source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst [hunk 4]
```diff
@@ -238,6 +238,11 @@ Issues Resolved
...
+   "EXT_SITMPUSW-29","TI Debian SDK: Docker not working"
```
Comment (AashvijShenai) [inline source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst:245]: Can you please add a space here at line 245

Comment (jsuhaas22) [reply source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst]: Done

**Review by praneethbajjuri — APPROVED** [review]

**Review by AashvijShenai — APPROVED** [review]

**Review by jeevantelukula — APPROVED** [review]

**Pull request merged**